sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubSharedAccessKeyCredential.java

```diff
@@ -28,6 +28,10 @@
 /**
  * Authorizes with Azure Event Hubs service using a shared access key from either an Event Hubs namespace or a specific
  * Event Hub.
+ *
+ * <p>
```

Comment:
Delete these empty `<p>` tags.

Reply:
Good catch. I wanted to add where to get the information to create this credential from. I'll update that.